src/Adapter/MSTest.TestAdapter/TestingPlatformAdapter/BridgedConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !WINDOWS_UWP
+using System.Diagnostics.CodeAnalysis;
+
+using Microsoft.Testing.Platform.Configurations;
+
+using PlatformServicesConfiguration = Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.IConfiguration;
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[SuppressMessage("ApiDesign", "RS0030:Do not use banned APIs", Justification = "We can use MTP from this folder")]
```

Comment:
This is not ideal, I would prefer to build a custom analyzer but wanted to ship a quick fix and this is "good enough" for now.